### PR TITLE
kinesisanalytics application: add tags support

### PR DESCRIPTION
**Pull request by raanand-dig**

### CODE REVIEW

**Comment by Shahard2** [comment]
@cristiangreco 
@kgeckhart 

Can you review it please?

File: pkg/config/services.go [hunk 1]
```diff
@@ -470,6 +470,12 @@ var SupportedServices = serviceConfigs{
 	{
 		Namespace: "AWS/KinesisAnalytics",
 		Alias:     "kinesis-analytics",
+		ResourceFilters: []*string{
+			aws.String("kinesisanalytics:application"),
```
[the remark this answers is not in the document]
Comment (raanand-dig) [reply pkg/config/services.go:474]: Yes

**Comment by Shahard2** [comment]
Thanks @cristiangreco !

Can you create a new release with it?

**Comment by cristiangreco** [comment]
@Shahard2 this will be included with the next release, but there are other changes to be merged first.

**Pull request merged**